Source/WebCore/animation/DocumentTimeline.cpp

```diff
@@ -83,6 +83,8 @@ AnimationTimelinesController* DocumentTimeline::controller() const
 
 void DocumentTimeline::detachFromDocument()
 {
+    Ref<DocumentTimeline> protectedThis(*this);
```

Comment:
I think the "modern" way to write this is `Ref protectedThis { *this };`

Reply:
Fixed.